server/src/db/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,23 @@
+import UserSerializer from "./UserSerializer.js"
+import { User } from "../../models/index.js"
+
+class ReviewSerializer {
+    static async showReviewDetails (reviewsArray) {
```

Comment:
nit picking - no space before the parameter parenthesis
https://github.com/airbnb/javascript#whitespace--around-keywords